tallycash/extension
# Persist activeNetwork on disk

Closes https://github.com/tallycash/extension/issues/1532

This prevents a bug where if a user has polygon selected as the wallet's network and closes chrome.  Next time they open it the UI will show polygon as the active network but the extension will in fact be connected to Ethereum.

### To test
- open wallet
- select polygon as network
- quit chrome
- open chrome again
- [ ] verify through devtools that the activeNetwork is in fact polygon and not ethereum
